### PR TITLE
fix: clarify how to create multiple box shadows (Issue: #35534)

### LINKED ISSUE #35534: Clarify how to create multiple box shadows
**Describe your problem and how to reproduce it:**
Two things:
- Make it easier to read the property values `box-shadow` can take
- Mention that multiple shadows are applied using commas

Currently, the list of possible values for `box-shadow` is messy, because it is within a paragraph and may be difficult to parse how many values a single shadow takes. Secondly, the example doesn't explain how there are multiple shadows. Turns out, this is done because there is a comma there.

**Suggested Fix**
- Mention that commas are used to create multiple box shadows
- Create a list of the values for `box-shadow`, something like this:

> The box-shadow property takes values for
> - `offset-x` (how far to push the shadow horizontally from the element),
> - `offset-y` (how far to push the shadow vertically from the element),
> - `blur-radius`,
> - `spread-radius` and
> - a color value, in that order.
>
> The `blur-radius` and `spread-radius` values are optional.

**Add a Link to the page with the problem:**
https://learn.freecodecamp.org/responsive-web-design/applied-visual-design/add-a-box-shadow-to-a-card-like-element

**Tell us about your browser and operating system:**
* Browser Name: Chrome
* Browser Version: Version 72.0.3626.117 (Official Build) (64-bit)
* Operating System: Chrome OS

**Note**: given the upcoming changes to [freeCodeCamp's next release](https://gitter.im/FreeCodeCamp/Contributors?at=5c7a23172ca5ec547462f85d) as of the writing of this issue, you may claim this issue to work on, but it being merged will probably happen after the next big release. If you have any doubts, [please ask in the contributors Gitter chat](https://gitter.im/FreeCodeCamp/Contributors).

### PR DESCRIPTION
**Pull request by shashank-bairy**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #35534 
Made the required changes asked by @erictleung to resolve issue #35534 . I have tested the changes locally and its working.

### CODE REVIEW

**Review by kyscg — APPROVED** [review]

File: curriculum/challenges/english/01-responsive-web-design/applied-visual-design/add-a-box-shadow-to-a-card-like-element.english.md [hunk 1]
```diff
@@ -8,7 +8,16 @@ videoUrl: 'https://scrimba.com/c/cvVZdUd'
...
 <section id='description'>
 The <code>box-shadow</code> property applies one or more shadows to an element.
-The <code>box-shadow</code> property takes values for <code>offset-x</code> (how far to push the shadow horizontally from the element), <code>offset-y</code> (how far to push the shadow vertically from the element), <code>blur-radius</code>, <code>spread-radius</code> and a color value, in that order. The <code>blur-radius</code> and <code>spread-radius</code> values are optional.
+The <code>box-shadow</code> property takes values for 
+<ul>
+  <li><code>offset-x</code> (how far to push the shadow horizontally from the element),</li>
+  <li><code>offset-y</code> (how far to push the shadow vertically from the element),</li>
+  <li><code>blur-radius</code>,</li>
+  <li><code>spread-radius</code> and</li>
+  <li>a color value, in that order.</li>
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/01-responsive-web-design/applied-visual-design/add-a-box-shadow-to-a-card-like-element.english.md]:
```suggestion
  <li><code>color</code>, in that order.</li>
```
I think this is more consistent with the rest of the bullet points.

**Review by ojeytonwilliams — COMMENTED** [review]
Other than that last line, this looks good to me.  Thanks for the PR!

**Comment by BA1RY** [comment]
Thanks for your suggestion @ojeytonwilliams . I have commited the changes you suggested.

**Comment by ojeytonwilliams** [comment]
@BA1RY Thank you for your contribution to the page! :+1:
We're happy to accept these changes, and look forward to future contributions. :memo:

**Comment by RandellDawson** [comment]
@ojeytonwilliams I sent you a DM on Gitter.

**Pull request merged**